docs/data/release_archive.yml

```diff
@@ -641,6 +641,11 @@ release_archive:
       version: 3.3.0
       release_date: "2024-10-17"
       filename: "kafka"
+    - name: "Flink AWS Connectors"
+      connector: "aws"
+      version: 5.0.0
+      release_date: "2024-11-07"
```

Comment:
Make sure to update this before you merge

Reply:
updated